Plugins/Mute/Commands/MuteCommand.cs

```diff
@@ -23,20 +26,31 @@ public class MuteCommand : Command
             {
                 Name = translationLookup["COMMANDS_ARGS_PLAYER"],
                 Required = true
+            },
+            new CommandArgument
+            {
+                Name = translationLookup["COMMANDS_ARGS_REASON"],
+                Required = true
             }
         };
     }
-
-    private readonly MuteManager _muteManager = new();
-
+    
     public override async Task ExecuteAsync(GameEvent gameEvent)
     {
-        if (await _muteManager.Mute(gameEvent))
+        var match = Regex.Match(gameEvent.Data, BanRegex);
```

Comment:
not sure why you're using regex here. 
Seems like all you're doing is just checking to make sure that the data has at least 1 character.
That's already handled by the command processing when the `Required` flag is true.